sched/jobDefSerializer.go

```diff
@@ -0,0 +1,112 @@
+package sched
+
+import (
+	"github.com/apache/thrift/lib/go/thrift"
+	"github.com/scootdev/scoot/sched/gen-go/sagalog"
+	"github.com/scootdev/scoot/runner"
+	"time"
+	"fmt"
+)
+
+const (
+	JsonSerialize string = "json"
+	BinarySerialize string = "binary"
```

Comment:
nit: running "go fmt" should create whitespace here to line up the two const declarations